scripts/jenkins-status.js

```diff
@@ -3,6 +3,21 @@
 const pushJenkinsUpdate = require('../lib/push-jenkins-update')
 const enabledRepos = ['citgm', 'node']
 
+const jenkinsIpWhitelist = process.env.JENKINS_WORKER_IPS ? process.env.JENKINS_WORKER_IPS.split(',') : []
+
+function isJenkinsIpWhitelisted (req) {
+  const ip = req.headers['x-forwarded-for'] || req.connection.remoteAddress
```

Comment:
Good question. I'm not totally confident it's actually needed no, would depend if there's some proxy in between the workers and the bot I assume.

Very good point that allows for side stepping the whole whitelist :/

If @jbergstroem don't have other thoughts on this, I'm okey with giving it a shot without using `x-forwarded-for`.